packages/gatsby-remark-responsive-iframe/src/index.js

```diff
@@ -61,17 +74,8 @@ module.exports = ({ markdownAST }, pluginOptions = {}) =>
           </div>
           `
 
-          node.data = {
-            hChildren: [{ type: `raw`, value: rawHTML }],
-          }
-          // Set type to unknown so mdast-util-to-hast will treat this node as a
-          // div not an iframe — it gets quite confused otherwise.
-          node.type = `unknown`
```

Comment:
@pieh do you have any insight into why this was done?

I validated the new approach with a sample project, and it works just fine setting the type to `html`.

Just entirely unsure if this would break other plugins. Doesn't look like we have other plugins depending on this in _our_ repo, at least.

Reply:
Sorry for late reply - I have absolutely no context on this. Maybe some older version of `mdast-util-to-hast` had problems with this? I vaguely remember that we did bump `mdast-util-to-hast` at some point, so it might fix the issue?

Reply:
No problem! I assumed as much--because I validated it worked like we expect with this new approach too, so hopefully no issues here!